### PR TITLE
update ir_cron.xml

### PR DESCRIPTION
**Pull request by nogueira-raphael**

### CODE REVIEW

File: saas_server_backup_ftp/models/res_config.py [hunk 1]
```diff
@@ -51,36 +51,31 @@ def get_values(self):
...
     def test_sftp_connection(self):
-        server = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_server", default=None)
-        username = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_username", default=None)
-        password = self.env["ir.config_parameter"].sudo().get_param(
-            "saas_server.sftp_password", default=None)
-        sftp_rsa_key_path = self.env["ir.config_parameter"].sudo(
-        ).get_param('saas_server.sftp_rsa_key_path')
+        server = self.env["ir.config_parameter"].get_param("saas_server.sftp_server", default=None)
```
Comment (njeudy) [inline saas_server_backup_ftp/models/res_config.py:54]: Pliz keep 80 chars max for line :)

**Review by njeudy — CHANGES_REQUESTED** [review]
pep8 and linter breaks if line is more than 80 chars

**Comment by njeudy** [comment]
@rodriguesraphael thanks for PR will merge when changes where done :)

File: saas_server_backup_ftp/models/res_config.py (hunk 1 above)
Comment (nogueira-raphael) [reply saas_server_backup_ftp/models/res_config.py:54]: But I kept 80 characters in the lines mentioned, incidentally, in the lines I mentioned I did not change anything.

**Comment by njeudy** [comment]
I change some info and push on my branch thanks for the work

**Pull request closed without merging**